components/globals/SecurityBadge.tsx

```diff
@@ -9,7 +9,11 @@ const SecurityAttributeBadge = (props: SecurityAttributeBadgeProps) => {
     : props.securityLevel + t("securityAttributeBadge.desc");
   return (
     <div className={props.className}>
-      <span className="gc-security-badge" aria-describedby="security-attribute-desc">
+      <span
+        id="security-attribute-content"
```

Comment:
If the `id` here is for testing only then I think you can use `data-testid` and then use it in the Cypress test.
(example: `cy.get("[data-testid='add-row-button-3']").click();`)